src/lib/logger.ts

```diff
@@ -0,0 +1,8 @@
+class Logger {
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  onMissingTranslation: (error: Error) => void = () => {};
```

Comment:
You might be able to satisfy the linter with
```suggestion
  onMissingTranslation: (error: Error) => void = () => null;
```

Reply:
done